src/components/atoms/Logo/index.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import './style.css';
+
+const Logo = () => {
+    return(
+        <h1 className="logo">Renome</h1>
```

Comment:
get data from file `structure.json`

Reply:
Created file data.json for storing the data

Reply:
pass all the data trough props, not directly from json and not in a hardcoded way